hw-probe.pl

```diff
@@ -217,15 +217,19 @@
 
 if($#ARGV_COPY==-1)
 { # Run from STDIN
-    print "Executing hw-probe -all -upload\n\n";
-    $Opt{"All"} = 1;
-    $Opt{"Upload"} = 1;
+    if (-t STDIN) {
+       	print "Executing hw-probe -all\n\n";
+        $Opt{"All"} = 1;
+    } else {
+       	print "Executing hw-probe -all -upload\n\n";
+       	$Opt{"All"} = 1;
+       	$Opt{"Upload"} = 1;
+    }
```

Comment:
Please fix alignment. Also please use spaces instead of tabs.

Reply:
Done.